mirgecom/fluid.py

```diff
@@ -0,0 +1,62 @@
+""":mod:`mirgecom.fluid` provides common utilities for fluid simulation.
+
+.. autofunction:: compute_local_velocity_gradient
+"""
+
+__copyright__ = """
+Copyright (C) 2020 University of Illinois Board of Trustees
+"""
+
+__license__ = """
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+"""
+import numpy as np  # noqa
+from meshmode.dof_array import DOFArray  # noqa
+from mirgecom.euler import ConservedVars
+
+
+def compute_local_velocity_gradient(discr, cv: ConservedVars):
+    r"""
+    Compute the cell-local gradient of fluid velocity.
+
+    Computes the cell-local gradient of fluid velocity from:
```

Comment:
Minor comment here! 
```suggestion
    Computes the cell-local gradient of fluid velocity from:
```

Reply:
The trouble is that the first sentence is required by pydocstyle to be one-line sentence in imperative mood.  The detailed description follows.

Reply:
Oh sorry I forgot about that. Nevermind then!